README.md

```diff
@@ -1,8 +1,6 @@
-# OPX Build
+# OPX build
 
-*Build environment and scripts for OpenSwitch.*
-
-If you would like to download binaries instead, see
+This repo is used to build environment and scripts for OpenSwitch. To download binaries instead, see
```

Comment:
- Repository instead of repo
- "Build environment" is an adjective and a noun, not a verb and a noun